components/script/dom/htmlinputelement.rs

```diff
@@ -1143,7 +1144,27 @@ impl VirtualMethods for HTMLInputElement {
 
 impl FormControl for HTMLInputElement {}
 
-impl Validatable for HTMLInputElement {}
+impl Validatable for HTMLInputElement {
+    fn is_instance_validatable(&self) -> bool {
+        match self.input_type.get() {
+            InputType::InputSubmit => {},
+            InputType::InputText => {},
+            InputType::InputReset => {},
+            InputType::InputButton => {},
+            InputType::InputFile => {},
+            InputType::InputImage => {},
+            InputType::InputCheckbox => {},
+            InputType::InputRadio => {},
+            InputType::InputPassword => {},
+        }
```

Comment:
We should either return real values for these types or remove the match until we actually use it.